<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="left_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>381</width>
      <height>571</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="current_period_name_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>current_period_name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="show_plan_button">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>360</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>show plan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="basic_state_record_button">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>420</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>记录基本状态</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="plan_text">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>341</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="right_frame">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>421</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="start_new_period_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>开启新时段</string>
     </property>
    </widget>
    <widget class="QPushButton" name="end_current_period_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>结束当前时段</string>
     </property>
    </widget>
    <widget class="QPushButton" name="close_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QListWidget" name="activity_list">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="current_activity_name_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>360</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>current_activity_name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="new_activity_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>410</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>新活动</string>
     </property>
    </widget>
    <widget class="QPushButton" name="end_activity_button">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>410</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>结束活动</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
